Sources/Amplitude/ObjC/ObjCAutocaptureOptions.swift

```diff
@@ -0,0 +1,100 @@
+import Foundation
+
+@objc(AMPAutocaptureOptions)
+public final class ObjCAutocaptureOptions: NSObject {
+    internal var _options: AutocaptureOptions
+
+    public override init() {
+        _options = AutocaptureOptions()
+        super.init()
+    }
+
+    @objc
+    public convenience init(optionsToUnion: [ObjCAutocaptureOptions]) {
+        self.init()
+        for option in optionsToUnion {
+            formUnion(option)
+        }
+    }
+
+    internal convenience init(options: AutocaptureOptions) {
+        self.init()
+        _options = options
+    }
+
+    internal var options: AutocaptureOptions {
+        get {
+            return _options
+        }
+        set {
+            _options = newValue
+        }
+    }
+
+    @objc
+    public static func sessions() -> ObjCAutocaptureOptions {
+        return ObjCAutocaptureOptions(options: .sessions)
+    }
+
+    @objc
+    public static func appLifecycles() -> ObjCAutocaptureOptions {
+        return ObjCAutocaptureOptions(options: .appLifecycles)
+    }
+
+    @objc
+    public static func screenViews() -> ObjCAutocaptureOptions {
+        return ObjCAutocaptureOptions(options: .screenViews)
+    }
+
+    @objc
+    public static func elementInteractions() -> ObjCAutocaptureOptions {
+        return ObjCAutocaptureOptions(options: .elementInteractions)
+    }
```

Comment:
any reason to have these as functions vs static lets?

Reply:
Nope, should be fixed in cb492547541cc15cc298ce18385e07f01202d9d4.